src/libs/CardUtils.ts

```diff
@@ -98,7 +98,7 @@ function isCardHiddenFromSearch(card: Card) {
     return !card?.nameValuePairs?.isVirtual && CONST.EXPENSIFY_CARD.HIDDEN_FROM_SEARCH_STATES.includes(card.state ?? 0);
 }
 
-function mergeCardListWithWorkspaceFeeds(workspaceFeeds: Record<string, WorkspaceCardsList | undefined>, cardList = allCards, shouldExcludeCardHiddenFromSearch = false) {
+function mergeCardListWithWorkspaceFeeds(workspaceFeeds: Record<string, WorkspaceCardsList | undefined> | undefined, cardList = allCards, shouldExcludeCardHiddenFromSearch = false) {
```

Comment:
IMO it looks bad when the first argument can be `undefined` but there are other arguments after it. It is technically "correct" from the perspective of JS/TS but bad function design.

Can we revert the change? did you do it just to avoid creating empty objects in other places in Search code?